cpp/create_module/algorithm/create.cpp

```diff
@@ -0,0 +1,47 @@
+#include "create.hpp"
+
+void Create::RemoveElementLabels(mgp::Node &element, const mgp::List &labels,
+                                 const mgp::RecordFactory &record_factory) {
+  for (auto label : labels) {
+    element.RemoveLabel(label.ValueString());
+    auto record = record_factory.NewRecord();
+    record.Insert(std::string(kResultRemoveLabels).c_str(), std::move(element));
+  }
+}
+
+void Create::ProcessElement(const mgp::Value &element, const mgp::Graph graph, const mgp::List &labels,
+                            const mgp::RecordFactory &record_factory) {
+  if (element.IsNode()) {
+    auto node = element.ValueNode();
+    RemoveElementLabels(node, labels, record_factory);
+    return;
+  }
+  if (element.IsInt()) {
```

Comment:
Can this int here be relationship id?

Reply:
No it can't, sorry I copied the exception message from other procedure and forgot to change it